app/assets/javascripts/devise/sessions/new.js

```diff
@@ -0,0 +1,20 @@
+$(".sessions.new").ready(function() {
```

Comment:
add selector to devise view and use it here, deduplicate logic with registrations.js